torchci/lib/drciUtils.ts

```diff
@@ -53,6 +53,12 @@ export const EXCLUDED_FROM_FLAKINESS = [
 // it increases the risk of getting misclassification. This guardrail can
 // be relaxed once we achieve better accuracy from the log classifier. This
 // sets the limit to 7 days
+
+export const ErrorsToNotDisable: RegExp[] = [
```

Comment:
```suggestion
export const LINE_CAPTURES_EXCLUDED_FROM_FLAKINESS: RegExp[] = [
```
Rename to something more specific?